mollie/api/resources/chargebacks.py

```diff
@@ -1,19 +1,28 @@
-from ..error import IdentifierError
 from ..objects.chargeback import Chargeback
 from .base import ResourceBase, ResourceGetMixin, ResourceListMixin
 
 
-class Chargebacks(ResourceBase, ResourceGetMixin, ResourceListMixin):
+class ChargebacksBase(ResourceBase):
     RESOURCE_ID_PREFIX = "chb_"
 
     def get_resource_object(self, result):
         return Chargeback(result, self.client)
 
-    def get(self, chargeback_id, **params):
-        """Verify the chargeback ID and retrieve the chargeback from the API."""
-        if not chargeback_id or not chargeback_id.startswith(self.RESOURCE_ID_PREFIX):
-            raise IdentifierError(
-                f"Invalid chargeback ID: '{chargeback_id}'. "
-                f"A chargeback ID should start with '{self.RESOURCE_ID_PREFIX}'."
-            )
+
+class Chargebacks(ChargebacksBase, ResourceListMixin):
+    pass
```

Comment:
For the class Refunds (resource/refunds.py) you've added a docstring. Could you do the same for this class?